### PR TITLE
Write logs to stdout

### PR DESCRIPTION
**Pull request by dleard**
- Turn on remote-logging
- Set values for elasticsearch
- pip install elasticsearch
- update postgres to use netapp-file instead of netapp-block (block kerplodes on openshift)

Docs: (under 'writing logs to elasticsearch')
https://github.com/apache/airflow/blob/1e3cdddcd87be3c0f11b43efea11cdbddaff4470/docs/howto/write-logs.rst

### CODE REVIEW

**Comment by dleard** [comment]
closes #7

File: postgresql/values.yaml [hunk 1]
```diff
@@ -64,7 +64,7 @@ persistence:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  storageClass: netapp-block-standard
+  storageClass: netapp-file-standard
```
Comment (wenzowski) [inline postgresql/values.yaml]: This is the only thing I want to get to the bottom of. I can take a poke at getting `netapp-block-standard` working again tomorrow.

**Review by wenzowski — CHANGES_REQUESTED** [review]
Only the one thing that I think is out of scope on this PR (may be necessary but I hope not). Happy to look into this tomorrow. Otherwise, looks great!

File: cas-airflow/values.yaml [hunk 2]
```diff
@@ -2,7 +2,13 @@ airflow:
...
     image:
       repository: docker-registry.default.svc:5000/wksv3k-tools/cas-airflow
-      tag: 0.0.1
+      tag: '0.1.0-pre.022'
+    config:
+      AIRFLOW__CORE__REMOTE_LOGGING: True
+      AIRFLOW__ELASTICSEARCH__HOST: 'airflow'
+      AIRFLOW__ELASTICSEARCH__LOG_ID_TEMPLATE: '{{dag_id}}-{{task_id}}-{{execution_date}}-{{try_number}}'
+      AIRFLOW__ELASTICSEARCH__END_OF_LOG_MARK: 'end_of_log'
+      AIRFLOW__ELASTICSEARCH__WRITE_STDOUT: 'true'
```
Comment (wenzowski) [inline cas-airflow/values.yaml:11]: `C_FORCE_ROOT` should be moved out of `extraEnv` too, right?

**Review by wenzowski — CHANGES_REQUESTED** [review]
I'm going to commit on your branch for a bit to try and resolve the netapp issue. Will comment here when handing back @dleard

File: cas-airflow/values.yaml (hunk 2 above)
Comment (dleard) [reply cas-airflow/values.yaml:11]: @wenzowski I don't think so? It sounds to me like C_FORCE_ROOT needs to be an env variable in order to run celery as root, whereas variables like AIRFLOW_ELASTICSEARCH_HOST are being inserted into the ariflow.cfg, which is why they are under airflow:config:

Comment (dleard) [reply cas-airflow/values.yaml:11]: https://stackoverflow.com/questions/20346851/running-celery-as-root

Comment (wenzowski) [reply cas-airflow/values.yaml:11]: Done mucking; though now wondering if this is the right approach. By writing logs to a fake elasticsearch we can't read them back. Maybe we should be writing them to an object storage bucket instead?

**Comment by wenzowski** [comment]
Done mucking; though now wondering if this is the right approach. By writing logs to a fake elasticsearch we can't read them back. Maybe we should be writing them to an object storage bucket instead?

**Comment by wenzowski** [comment]
A look at the persistent volume claim annotations revealed that volume provisioning was stuck. Deleting the claim and re-provisioning solved the issue.
```
oc -n wksv3k-tools get pvc cas-airflow-postgresql -o json | jq '.metadata.annotations'
```

**Pull request closed without merging**